Source/DafnyCore/AST/Modules/ModuleDefinition.cs

```diff
@@ -321,6 +321,14 @@ public static IEnumerable<Field> AllFields(IEnumerable<TopLevelDecl> declaration
     }
   }
 
+  public static IEnumerable<MemberDecl> AllMembers(IEnumerable<TopLevelDecl> declarations) {
+    foreach (var decl in declarations.OfType<TopLevelDeclWithMembers>()) {
```

Comment:
better to use `declarations.OfType<TopLevelDeclWithMembers>().SelectMany(d => d.Members)`